k8s/regions/oregon/prod.yaml

```diff
@@ -0,0 +1,74 @@
+# General settings
+TARGET_ENVIRONMENT: prod
+AWS_REGION: us-west-2
+AWS_RESOURCE_STACK: SUMO-prod
```

Comment:
Are those two AWS_* variables used somewhere?

Reply:
doesn't look like it, it was carried over from MDN automation.

Reply:
removed in https://github.com/mozilla/kitsune/pull/3085